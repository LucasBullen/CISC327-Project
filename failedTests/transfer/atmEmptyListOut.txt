1,7c1,4
< Select mode: atm or agent.
< Logged into atm mode.
< Enter first account number.
< Enter second account number.
< Incorrect account number format (8 digits, no leading zeros). Transfer cancelled.
< Unrecognized command. Remember all commands must be in lowercase.
< Unrecognized command. Remember all commands must be in lowercase.
---
> Select mode: atm or agent.
> Logged into atm mode.
> Enter number of account to transfer from.
> Logged out.
