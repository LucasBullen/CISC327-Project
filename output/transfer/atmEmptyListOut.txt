Select mode: atm or agent.
Logged into atm mode.
 Enter number of account to deposit into.
Account does not exist. Transfer cancelled.
Logged out.
