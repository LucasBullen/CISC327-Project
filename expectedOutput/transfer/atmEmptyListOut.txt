Select mode: atm or agent.
Logged into atm mode.
Enter number of account to transfer from.
Logged out.
